tests/components/plex/payloads.py

```diff
@@ -0,0 +1,828 @@
+"""Plex XML payloads for testing.
+
+Calls to xml.etree.ElementTree.fromstring are allowed as all payloads are static and known.
+"""
+import xml.etree.ElementTree as ET
+
+
+def media_container(attribs={}, *payloads):
+    """Create a MediaContainer XML wrapper with the provided attributes."""
+    mc = ET.Element("MediaContainer", attrib=attribs)
+    mc.set("size", str(len(payloads)))
+    for payload in payloads:
+        mc.append(payload)
+    return ET.tostring(mc, encoding="unicode")
+
+
+EMPTY_PAYLOAD = media_container()
```

Comment:
Is it impossible to store the xml fixtures as xml files and load when needed? It would be nice to avoid the logic that is present in this module.

Reply:
Not impossible, but something I didn't want to do. There's a lot of redundancy across the payloads and it was taking considerable effort to sanitize & normalize the payloads taken from a "real" installation to be used here. For example, remembering to edit the same UUID across a half-dozen payloads to keep them in sync was becoming unmanageable.

Reply:
We don't want logic that describes device specific info in home assistant. This is the same standard as why we don't have device specific code in production but require a library.

Loading an xml file is ok for the tests.

Reply:
Finally got around to changing this. Is this what you had in mind?